source/includes/plugin_info/_010-pluggable-instance-settings.md.erb

```diff
@@ -1,26 +1,171 @@
 <%
-    json = {
-  pluggable_instance_settings: {
+    package_settings_json = {
+  package_settings: {
+    configurations: [
+      {
+        key: "Package_name",
+        metadata: {
+          secure: false,
+          required: true,
+          part_of_identity: false,
+          display_name: "Display name",
+          display_order: 0
+        }
+      }]
+  }
+}
+%>
+
+<%=
+describe_sub_object 'Package settings object', json: package_settings_json, html_id: 'package-instance-settings' do
+  configurations    'Array', 'List of [configurations](#package-settings-configurations) required to configure a pluggable instance'
+end
+%>
+
+<%
+    repository_settings_json = {
+  repository_settings: {
+    configurations: [
+      {
+        key: "url",
+        metadata: {
+          secure: false,
+          required: true,
+          part_of_identity: false,
+          display_name: "Display name",
+          display_order: 0
+        }
+      }]
+  }
+}
+%>
+
+<%=
+describe_sub_object 'Repository settings object', json: repository_settings_json, html_id: 'repository-instance-settings' do
+  configurations    'Array', 'List of [configurations](#repository-settings-configurations) required to configure a pluggable instance'
+end
+%>
+
+<%
+    scm_settings_json = {
+  scm_settings: {
+    configurations: [
+      {
+        key: "url",
+        metadata: {
+          secure: false,
+          required: true,
+          part_of_identity: false,
+        }
+      }],
+    views: {
+      template: "<div>Plugin view template</div>"
+    }
+  }
+}
+%>
+
+<%=
+describe_sub_object 'SCM settings object', json: scm_settings_json, html_id: 'scm-instance-settings' do
+  configurations    'Array', 'List of [configurations](#scm-settings-configurations) required to configure a pluggable instance'
+  views             'Object', 'The [view](#plugin-info-view) contains the Angular view template'
+end
+%>
+
+<%
+    plugin_settings_json = {
+  plugin_settings: {
     configurations: [
       {
         key: "url",
         metadata: {
           secure: false,
           required: true,
-          part_of_identity: false
+          part_of_identity: false,
```

Comment:
I think plugin_settings won't have `part_of_identity` as per the new PluginInfoBuilder. https://github.com/gocd/gocd/blob/master/plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/common/PluginInfoBuilder.java#L39